internal/desktop_browser.py

```diff
@@ -365,6 +367,19 @@ def on_start_recording(self, task):
                     self.execute_js(SET_ORANGE)
                     time.sleep(1)
                 task['video_file'] = os.path.join(task['dir'], task['prefix']) + '_video.mp4'
+                if platform.system() == 'Windows':
+                    from win32api import GetSystemMetrics
+                    self.screen_width = GetSystemMetrics(0)
+                    self.screen_height = GetSystemMetrics(1)
+                elif platform.system() == 'Darwin':
+                    from AppKit import NSScreen
+                    self.screen_width = int(NSScreen.screens()[0].frame().size.width)
+                    self.screen_height = int(NSScreen.screens()[0].frame().size.height)
+                else:
+                    self.screen_width = 1920
+                    self.screen_height = 1200
+                task['width'] = self.screen_width if task['width'] > self.screen_width else task['width']
```

Comment:
A bit easier to read if this is changed to min(self.screen_width, task['width']) (and for height).